web/src/main/java/org/cbioportal/genome_nexus/web/mixin/StatsByTumorTypeMixin.java

```diff
@@ -26,8 +26,11 @@
     private Double msiScore;
 
     @ApiModelProperty("Number With Signature")
```

Comment:
This should be ideally `Number of Variants with Signature` or something like that, right? But I guess we can think about it later.